README.md

```diff
@@ -63,6 +63,13 @@ When deploying the multi cluster traffic controller using the make targets the f
     make local_setup MCTC_WORKLOAD_CLUSTERS_COUNT=<NUMBER_WORKLOAD_CLUSTER>
     ```
 
+1. To ensure the webhook server is reachable from the workload cluster. Deploy
+   the webhook proxy
+
+   ```sh
+   make webhook-proxy
+   ```
```

Comment:
This would be better being installed as part of the local-setup script since it's intended purpose is to setup a dev env locally using kind. Presumably though we don't actually want this deployed if you are actually running the controller on the cluster? If just had the proxy listen on a different port from the default(8082) could we not just install this all the time on the kind control cluster?

Reply:
I thought of it but, as you say, the main reason against it is the case that we'd want to run the controller in-cluster.
The problem of using a different port is that in that case we'd have to go and modify the port in the webhook configuration if we want to run it in cluster.

I haven't tested it, but I think it might still be possible to deploy the proxy by default in the local-setup and, if running `make deploy`, it would update the service to point to the controller instead of the proxy, so it would still work without any manual changes

Reply:
I just tested it and it does work, running `make deploy` after the proxy has been deployed. It configures the service to point to the controller, and everything else can remain the same. The command just fails to create the ingress but that's ok because the spec is the same as the proxy. I'm going to figure out if there's a way to not fail the command and include the deployment of the proxy in the local-setup. How does that sound?